File: src/main/java/xyz/chalky/taboo/commands/misc/InfoSlashCommand.java
package xyz.chalky.taboo.commands.misc;

import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
import net.dv8tion.jda.api.interactions.commands.build.Commands;
import xyz.chalky.taboo.backend.SlashCommand;

public class InfoSlashCommand extends SlashCommand {

    public InfoSlashCommand() {
        setCommandData(Commands.slash("info", "Displays information about me!"));
    }

    @Override
    public void executeCommand(SlashCommandInteractionEvent event) {
        event.getHook().sendMessage("Hi! I'm Taboo!").queue();
    }

}

File: src/main/kotlin/xyz/chalky/taboo/commands/misc/InfoSlashCommand.kt
package xyz.chalky.taboo.commands.misc

import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent
import net.dv8tion.jda.api.interactions.commands.build.Commands
import xyz.chalky.taboo.backend.SlashCommand
import xyz.chalky.taboo.util._reply

class InfoSlashCommand : SlashCommand() {

    init {
        setCommandData(Commands.slash("info", "Displays information about me!"))
    }

    override fun executeCommand(event: SlashCommandInteractionEvent) {
        event._reply("Hi! I'm Taboo!").queue()
    }

}